app/controllers/concerns/authorization.rb

```diff
@@ -2,26 +2,25 @@ module Authorization
   extend ActiveSupport::Concern
 
   included do
+    helper_method :unauthorized!
     helper_method :authorize_super_admin!
     helper_method :authorize_admin!
-    helper_method :authorize_deployer!
-    helper_method :unauthorized!
-  end
-
-  def authorize_super_admin!
-    unauthorized! unless current_user.is_super_admin?
   end
 
-  def authorize_admin!
-    unauthorized! unless current_user.is_admin?
+  def unauthorized!
+    # Eventually to UnauthorizedController
+    throw(:warden)
   end
 
   def authorize_deployer!
     unauthorized! unless current_user.is_deployer?
   end
 
-  def unauthorized!
-    # Eventually to UnauthorizedController
-    throw(:warden)
+  def authorize_super_admin!
+    unauthorized! unless current_user.is_super_admin?
+  end
+
+  def authorize_admin!
+    unauthorized! unless current_user.is_admin?
```

Comment:
Re-ordering lines of code makes the diff harder because Git can't figure out what actually got added and not just moved. 

In this case, I think you just removed the 2 helper_methods?